cumulusci/cli/options.py

```diff
@@ -0,0 +1,28 @@
+"""Process environment variables and global configurations for some options.
+
+Returns option values in the following order of precedence:
+1. Global configuration options
+2. Click Option
+3. Environment variables
+4. Click Option default value
+"""
+
+
+from cumulusci.core.config import BaseGlobalConfig
+
+# CLI Environment Variables
+CCI_ENV_PREFIX = "CCI__"
+CLI_ENV_PREFIX = CCI_ENV_PREFIX + "CLI__"
+NO_PROMPT_ENV = CLI_ENV_PREFIX + "ALWAYS_RECREATE"
+PLAIN_OUTPUT_ENV = CLI_ENV_PREFIX + "PLAIN_OUTPUT"
```

Comment:
The config setting is cli__plain_output so I think this needs to be "CCI__CLI__PLAIN_OUTPUT"